parsers/to-style-dictionary/README.md

```diff
@@ -0,0 +1,457 @@
+# To Style Dictionary
+
+## Description
+
+Format design tokens to create a theme compatible with the [Style Dictionary specification](https://amzn.github.io/style-dictionary/#/).
+
+## Interface
+
+```ts
+interface parser {
+  name: 'to-style-dictionary';
+  options: Partial<{
+    formatName: 'camelCase' | 'kebabCase' | 'snakeCase' | 'pascalCase';
+    formatTokens: Partial<{
+      colorFormat: {
+        format: ColorsFormat;
+      };
+      fontSizeFormat: {
+        unit: 'px' | 'rem' | 'none';
+      };
+      fontFormat: Array<'woff2' | 'woff' | 'otf' | 'ttf' | 'eot'>;
+    }>;
+    splitBy?: string;
+    assetsBaseDirectory?: Partial<{
+      fonts?: string;
+      images?: string;
+      icons?: string;
+    }>;
+    formatConfig?: Partial<{
+      endOfLine: 'auto' | 'lf' | 'crlf' | 'cr';
+      tabWidth: number;
+      useTabs: boolean;
+    }>;
+  }>;
+}
+```
+
+### Options
+
+| Parameter                          | Required | Type                                                              | Default     | Description                                                                                                                                                                                                |
+| ---------------------------------- | -------- | ----------------------------------------------------------------- | ----------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `formatName`                       | optional | `camelCase` `kebabCase` `snakeCase` `pascalCase`                  | `camelCase` | The case transformation you want to apply to your design token name                                                                                                                                        |
```

Comment:
Please start all descriptions with an uppercase letter and add a "." at the end of all of them.